### ISSUE TITLE
Window title/Windows taskbar button text changed with Chrome v96

### ISSUE THREAD

**Issue by erikgeurts** [leggett/simplify#627]
After Chrome was updated to v96 (on a Windows 10 PC), I noticed a change in the window title, when launching Gmail as a window from a shortcut.

Context: I've created a shortcut for my Gmail by using the "More tools" -> "Create shortcut..." option. I've named the shortcut "Gmail prive" (privé is Dutch for personal). Double clicking that shortcut opens Gmail in a window.
![image](https://user-images.githubusercontent.com/5102461/142428733-18dd7ff7-05e2-4313-9802-cc74bcc65957.png)

Before Chrome v96, that window, and therefore the corresponding button on the Windows taskbar, used to show something like: "Inbox (2) - first.last@gmail.com - Gmail" which was abbreviated to "Inbox (2) - first..." or something similar.

Now with Chrome v96, the title suddenly changed to "Gmail prive - Inbox (2) - first.last@gmail.com - Gmail". That means that on the button all I see is "Gmail prive - Inbox.."
![image](https://user-images.githubusercontent.com/5102461/142428541-aabcc35d-0431-45e1-ab59-d43ecd0e9af6.png)

I searched the release notes of Chrome 96 to find out if there's a setting that affects this, but I couldn't find it. Perhaps I was looking for the wrong terminology.

Not being able to see the unread count in the taskbar button is a shame. I was wondering if there's anything you could do with Simplify to get rid of that addition of the shortcut name at the start of the window/button title.

**Comment by leggett** [comment]
This would be hard to detect and remove without also messing up labels that have a " - " in their name. 

I assume you named it "Gmail prive" and that wasn't some default I can assume? If you rename the shortcut to "Gmail" does it change the title?

Also, is that supposed to be "private" in French?

**Comment by erikgeurts** [comment]
Yes, Private in French is the equivalent of Prive in Dutch, It's actually incorrectly spelled, but I didn't enter the correct diacritical character because I wasn't sure Windows or Chrome would like that...

I entered the name myself while creating the shortcut, it's not the default. This is what is suggested when you're about to make a shortcut:
![image](https://user-images.githubusercontent.com/5102461/142454270-d57e615f-eebc-4220-8027-b7bd475537f4.png)

And then this is what I did:
![image](https://user-images.githubusercontent.com/5102461/142454410-14d1eb8f-3e16-4548-bc75-375371aebc28.png)

I also have similar shortcuts for multiple business G Suite accounts for companies I work for, hence the need to have a number of shortcuts all with a different name.

I tried renaming the shortcut on my desktop, but the window title doesn't change. May be caused by browser caching, but it could also be that name of the shortcut is not tied to the name of the "application" within the context of Windows and Chrome.

I really wonder why the Chrome developers made this change, and if there is perhaps a setting that I could adjust to change this. So far I haven't found that yet, but if anyone knows of a way to make that happen, please let me know.

**Comment by leggett** [comment]
Got it. I'll see what I can do when I can. 

I bet the desktop shortcut doesn't change the name used in the title (probably in a manifest file or something and set when you created the shortcut). You might be able to use this to fix the issue.

One crazy idea: 

- Delete the shortcut
- Save a new shortcut and name it something short like "G1"
- Then rename the shortcut on your desktop to "Gmail prive"
- Profit? (Is the title now just "G1 - Inbox..."

**Comment by erikgeurts** [comment]
I'm sure this affects many people using any other site or application, not just Gmail, with this type of shortcut. I'm struggling to understand why the Chrome developers thought this would be a good move...
